components/tag/selectableList/src/index.js

```diff
@@ -0,0 +1,109 @@
+import React, {Component, PropTypes} from 'react'
+import TagSelectable from '@schibstedspain/sui-tag-selectable'
+
+const removeFromArray = (arr, value) => {
+  return arr.filter(item => value !== item)
+}
+
+export default class TagSelectableList extends Component {
+  state = {
+    selectedValues: []
+  }
+
+  constructor (...props) {
+    super(...props)
+    this.tagHandlers = this.props.tagsList.map((tag) => this.toggleValue.bind(this, tag.value))
+    this.allHandler = this.toggleAll.bind(this)
```

Comment:
You can avoid this if you use an arrow function in the class method `toggleAll` ;)